README.md

```diff
@@ -0,0 +1,73 @@
+# XHarness
+
+This repo contains the code to build the **XHarness dotnet tool**.
+
+## What is XHarness
+
+XHarness is primarily a command line tool that enables running tests on Android and iOS (WatchOS and tvOS are also supported).
+It can locate devices/emulators, install given application, run it and collect results (uninstall it after).
```

Comment:
grammar nits: "install a given application", "uninstalling it after" or "and uninstall it after"